### PR TITLE
New type definitions for npm package 'unzip'

### PR DESCRIPTION
**Pull request by ravihugo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/unzip/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Type definitions for unzip 0.1
+// Project: https://github.com/EvanOxfeld/node-unzip
+// Definitions by: Ravi L. <https://github.com/coding2012>
```
Comment (sandersn) [inline types/unzip/index.d.ts]: Ravi L. (coding2012) or ravihugo?

File: types/unzip/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export interface Entry {
+    path: string;
+    type: 'Directory' | 'File';
+    size: number;
+    pipe: (stream: NodeJS.WritableStream) => void;
+    autodrain: () => void;
+}
+
+/*~ You can declare properties of the module using const, let, or var */
+export const myField: number;
```
Comment (sandersn) [inline types/unzip/index.d.ts]: remove example code

**Review by sandersn — CHANGES_REQUESTED** [review]
Need of couple of housekeeping items addressed before merging.

File: types/unzip/index.d.ts (hunk 1 above)
Comment (ravihugo) [reply types/unzip/index.d.ts]: Not sure what I'm supposed to change here, I would like to keep it under coding2012 if possible.

Comment (sandersn) [reply types/unzip/index.d.ts]: Just wanted to make sure the mismatch was intentional. 👍

**Review by sandersn — APPROVED** [review]

**Pull request merged**